ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/__init__.py

```diff
@@ -0,0 +1,76 @@
+from ansible_collections.arista.avd.plugins.plugin_utils.avdfacts import AvdFacts
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.interface_descriptions import load_interfacedescriptions
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.ip_addressing import load_ip_addressing
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.eos_cli import EosCliMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.ethernet_interfaces import EthernetInterfacesMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.ip_igmp_snooping import IpIgmpSnoopingMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.ip_virtual_router_mac_address import IpVirtualRouterMacAddressMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.ipv6_static_routes import Ipv6StaticRoutesMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.loopback_interfaces import LoopbackInterfacesMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.patch_panel import PatchPanelMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.port_channel_interfaces import PortChannelInterfacesMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.prefix_lists import PrefixListsMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.route_maps import RouteMapsMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.router_bgp import RouterBgpMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.router_isis import RouterIsisMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.router_ospf import RouterOspfMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.static_routes import StaticRoutesMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.struct_cfg import StructCfgMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.virtual_source_nat_vrfs import VirtualSourceNatVrfsMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.vlan_interfaces import VlanInterfacesMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.vlans import VlansMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.vrfs import VrfsMixin
+from ansible_collections.arista.avd.roles.eos_designs.python_modules.network_services.vxlan_interface import VxlanInterfaceMixin
+
+
+class AvdStructuredConfig(
```

Comment:
If there is no issue I would reorder the Mixins in alphabetical order

Reply:
It will reorder the output. Not that it matters much, but it will just produce much more changes to the structured config. (the order of root keys).